src/markdown/sanitize-schema.ts

```diff
@@ -1,9 +1,8 @@
+import deepmerge from "deepmerge"
```

Comment:
I think maybe `structuredClone` can be used as a replacement for it.

Reply:
`structuredClone` should be used to obtain a copy of the object?
Here it is expected to merge the properties of the object so that we do not need many levels of `...`.

Reply:
[structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) is a native deep cloning function, so we don't need to introduce a new dependency library.

Reply:
Sorry, I may not have expressed myself well. This is not about getting a deep copy, but to merge custom properties and default values.

I use it to make the following writing more concise.

```ts
import { defaultSchema } from "rehype-sanitize"

const scheme = {
  ...defaultSchema,
  attributes: {
    ...defaultSchema.attributes,
    "*": [...(defaultSchema.attributes?.["*"] || []), "className", "style"],
    video: ["src", "controls", "loop", "muted", "autoPlay", "playsInline"],
  protocols: {
    ...defaultSchema.protocols,
    href: [...(defaultSchema.protocols?.href ?? []), ...["magnet", "ed2k"]],
  },
}

export default scheme
```

Reply:
Or do you mistook `deepmerge` for `deepclone`?

Reply:
oh sorry, I got confused.